framework/test/jest/integration/specs/modules/chain/components/storage/entities/Account/get.spec.js

```diff
@@ -0,0 +1,187 @@
+const {
+	Account,
+} = require('../../../../../../../../../../src/modules/chain/components/storage/entities');
+const { PgHelper } = require('../../../../../../../utils/pg-helper');
+
+describe('storage.entities.Account.get', () => {
+	let pgHelper;
+	let storage;
+	let db;
+
+	beforeAll(async () => {
+		// Arrange
+		pgHelper = new PgHelper({ dbName: 'AccountGet' });
+
+		// Create second postgres connection
+		db = await pgHelper.bootstrap();
+
+		// Setup storage for Accounts
+		storage = await pgHelper.createStorage();
+		storage.registerEntity('Account', Account);
+
+		storage.entities.Account.extendDefaultOptions({
+			limit: 101, // @todo get it from constants
+		});
+	});
+
+	afterAll(async () => {
+		await db.done();
+		await pgHelper.cleanup();
+	});
+
+	beforeEach(async () => {
+		await db.query('DELETE FROM mem_accounts');
+	});
+
+	describe('Given arguments ({publicKey}, {extended: true}, tx)', () => {
+		it('should return array that contains 1 extended account object', async () => {
+			// Arrange
+			const account = {
+				address: 'delegateAddress',
+				isDelegate: 0,
+				publicKey:
+					'399a7d14610c4da8800ed929fc6a05133deb8fbac8403dec93226e96fa7590ee',
+				balance: 55566,
+			};
+
+			const expectedAccount = {
```

Comment:
Can we maybe extract this expected account object to a separate object as it is also reused in the test below. Something like:
```
const baseAcc= {...}

// Test
const expectedAccount = {
   ...account,
   ...baseAcc,
}
```

Reply:
With #4270 I'm moving those objects to separate folder anyways. So please ignore them in this pr.